drivers/wifi/nrf700x/src/fmac_main.c

```diff
@@ -71,7 +71,11 @@ BUILD_ASSERT(CONFIG_NRF700X_TX_MAX_DATA_SIZE >= MAX_TX_FRAME_SIZE,
 	"TX buffer size must be at least as big as the MTU and headroom");
 
 static const unsigned char aggregation = 1;
+#ifdef CONFIG_NRF_WIFI_WMM_MODE
 static const unsigned char wmm = 1;
```

Comment:
Why do we need this variable any longer? `CONFIG_NRF_WIFI_WMM_MODE` could just be used to set the value on line 700 ?